object/service.proto

```diff
@@ -116,9 +116,9 @@ service ObjectService {
   //   provided session token has expired.
   rpc Search(SearchRequest) returns (stream SearchResponse);
 
-  // Get byte range of data payload. Range is set as an (offset, length) tuple.
+  // Get byte range of data payload. Range is set as a (offset, length) tuple.
```

Comment:
Here the '()' are defining the tuple, so `a` comes right before `o` and should be transformed into `an`, isn't it?

Reply:
Well, I would not consider anything in brackets but check what stands after them, like `a ... tuple`.
https://www.quora.com/In-English-grammar-should-the-word-before-the-parenthesis-be-an-or-a-ie-do-you-match-it-to-the-first-word-in-the-bracket-or-first-word-after-the-closing-bracket